src/olympia/reviewers/serializers.py

```diff
@@ -174,6 +177,17 @@ def _fetch_entries():
 
         return self._entries
 
+    def get_mimetype(self, obj):
+        entries = self.get_entries(obj)
```

Comment:
For https://github.com/mozilla/addons-server/issues/14020 to work and help performance we'll want to avoid fetching the entries when fetching a single file, right ? This would mean this and even the `get_selected_file()` implementation would need to change to stop going through all entries every time we request a single file (it could be in cache, since the file tree is likely to be requested in parallel by code manager, but it could also not if we're unlucky)

Reply:
Yes, there will be more changes needed in this area. I started a big patch and then decided to break it down into smaller pieces, and this is just the first small piece. 

I had thought that we might be able to get away with keeping this logic, because code manager will never actually just need a file without also needing the version information the first time, which would then cause this data to be cached, but then I realized that we are pre-fetching the "next" file on each page, and therefore issuing two requests in parallel (one would be for the version + file, and the other would be for just the file), and both of those could theoretically kick off the process to populate the cache.

Reply:
The other helpful thing about this small patch is that it doesn't require any changes to code manager, whereas subsequent patches will need to be landed in tandem with patches in code manager.

Reply:
Yeah I'm fine with that approach as long as you've got improvements lined up for later.